projects/plugins/jetpack/modules/shortcodes/twitter.php

```diff
@@ -0,0 +1,70 @@
+<?php // phpcs:ignore
```

Comment:
Nitpick, but instead of the phpcs:ignore here, what do you think about adding the file doc comment that phpcs complains about? That would be an opportunity to explain:

- Why this is here and what it's used for (especially considering we already have a `tweet.php` file in the same directory).
- When it was introduced (with `@since $$next-version$$`)
- How it is loaded (it's not a typical shortcode, it will be loaded even when you don't use the module, because it gets force-loaded via `module-extras.php`)

Reply:
I've pushed a change that should resolve the file doc comment. Thank you for the feedback.